examples/playwright/src/tests/theia-terminal-view.test.ts

```diff
@@ -55,6 +55,7 @@ test.describe('Theia Terminal View', () => {
 
         // close all terminals
         for (const terminal of allTerminals) {
+            await terminal.activate(); // close box is not visible when not active
```

Comment:
minor: do we mean the close icon? I'm not aware of the "close box" which makes the documentation confusing.

Reply:
I think the term "close box" is quite common for windows (at least Google thinks so). I had "close button" before, but the UI element is not really a button. An "Icon" feels more like describing the implementation, not the UI element, i.e. the "close icon" is the graphic element, whereas Windows used to have a minus-sign in its window close box. 
If you still find it confusing, I can change it.